static/app/views/projectInstall/platformOrIntegration.tsx

```diff
@@ -1,28 +1,72 @@
-import * as qs from 'query-string';
+import {useState} from 'react';
+import type {RouteComponentProps} from 'react-router';
 
 import {OnboardingContextProvider} from 'sentry/components/onboarding/onboardingContext';
+import platforms from 'sentry/data/platforms';
+import {t} from 'sentry/locale';
+import type {PlatformIntegration} from 'sentry/types/project';
 import {platformToIntegrationMap} from 'sentry/utils/integrationUtil';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
 
 import GettingStarted from './gettingStarted';
 import {ProjectInstallPlatform} from './platform';
 import PlatformIntegrationSetup from './platformIntegrationSetup';
 
-type Props = React.ComponentProps<typeof ProjectInstallPlatform> &
-  Omit<React.ComponentProps<typeof PlatformIntegrationSetup>, 'integrationSlug'>;
-
-function PlatformOrIntegration(props: Props) {
-  const parsed = qs.parse(window.location.search);
-  const {platform} = props.params;
-  const integrationSlug = platform && platformToIntegrationMap[platform];
-  // check for manual override query param
-  // TODO(priscila): check this case
-  if (integrationSlug && parsed.manual !== '1') {
-    return <PlatformIntegrationSetup integrationSlug={integrationSlug} {...props} />;
+const allPlatforms: PlatformIntegration[] = [
+  ...platforms,
+  {
+    id: 'other',
+    name: t('Other'),
+    link: 'https://docs.sentry.io/platforms/',
+    type: 'language',
+    language: 'other',
+  },
+];
```

Comment:
`platforms` already includes the `other` platform.

Reply:
good catch!